Gemfile

```diff
@@ -8,6 +8,7 @@ gem 'rubocop', '~> 0.42.0', group: :lint
 gem 'chefspec', '~> 4.4.0', group: :unit
 gem 'mysql2', '~> 0.4.4', group: :unit
 gem 'hashie', '<= 3.4.4', group: :unit
+gem 'nio4r', '<= 1.2.1', group: :unit
```

Comment:
Otherwise Travis tests won't run due to old ruby version.
https://travis-ci.org/abiquo/abiquo-cookbook/builds/190641593